build/github/build-and-publish-probes-for-operating-system/main.go

```diff
@@ -191,6 +210,48 @@ func getFalcoDrivers(dockerCli *docker.Client, FalcoVersionNames []string) ([]fa
 	return FalcoVersions, nil
 }
 
+func falcoVersionHasRelease(releases []*github.RepositoryRelease, falcoVersions []falcoVersion) bool {
```

Comment:
nit: could we put this func in `//pkg/repository/ghreleases`? We're breaking our separation of concerns with it here as now this `main.go` layer is implementing logic specific to GitHub releases. I'd only expect us to use `"github.com/google/go-github/v37/github"` inside `pkg/repository/ghreleases`